apps/oidnTest.cpp

```diff
@@ -57,35 +57,43 @@ TEST_CASE("single filter", "[single_filter]")
 {
   const int W = 257;
   const int H = 89;
+  
+  std::vector<DeviceType> deviceType = { DeviceType::CPU };
+#if defined(OIDN_MPS)
+  deviceType.push_back(DeviceType::MPS);
+#endif
+  
```

Comment:
Aborting review. Too many undocumented general changes (outside of a #if defined(OIDN_MPS))